Libraries/LibGfx/PaintStyle.h

```diff
@@ -162,6 +163,57 @@ class CanvasRadialGradientPaintStyle : public GradientPaintStyle {
     float m_end_radius { 0.0f };
 };
 
+class SVGPatternPaintStyle final : public PaintStyle {
+public:
+    enum class PatternUnits {
```

Comment:
If you define these enums as u8 (eg, `enum class PatternUnits : u8 {`) they only take up a byte each instead of 4 bytes each.